example/provider-extensions/kyverno/kustomization.yaml

```
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://raw.githubusercontent.com/kyverno/kyverno/release-1.7/config/release/install.yaml
+- https://github.com/kyverno/kyverno/releases/download/v1.8.0/install.yaml
```

```suggestion
- https://github.com/kyverno/kyverno/releases/download/v1.9.5/install.yaml
```
Could we update directly to `v1.9.5`?
According to the changelog([ref](https://github.com/kyverno/kyverno/releases/tag/v1.9.0)), it supports Kubernetes v1.26 too.
Additionally, its `kyverno-verify-mutating-webhook-cfg` seems to be not problematic even when https://github.com/gardener/gardener/pull/8034 was not applied to the landscape yet.

---

> Additionally, its kyverno-verify-mutating-webhook-cfg seems to be not problematic even when https://github.com/gardener/gardener/pull/8034 was not applied to the landscape yet.

No, it is problematic, On master try creating as shoot using `kind-extesnion-setup` and the shoot used for this setup will get into the error with following error -
```
MutatingWebhookConfiguration "kyverno-verify-mutating-webhook-cfg" is problematic and was remediated by Gardener (please check its annotations for details).
```

---

I meant when you are using kyverno v1.9.5, otherwise it is problematic.

---

Ok, yes it will not be problematic, kyverno added `objectSelector` for `kyverno-verify-mutating-webhook-cfg` webhook v1.8.0 onwards.

---

Someone updated the landscape while I was upgrading from kyverno `v1.8.0` to `v1.9.5`. That's why I thought `v1.9.5` even works without #8034. But they behave the same way in regards of the webhook 😅